### ISSUE TITLE
Crd's Additional Service Endpoint Is Matching By VMServicescrape Selectors By Default

### ISSUE THREAD

**Issue by mohammadkhavari** [VictoriaMetrics/operator#578]
I have set serviceSpec of vmauth:
 ```
spec:
 serviceSpec:
    …
    spec:
      type: LoadBalancer
```
and thus there are two endpoints with following labels:
```
vmauth-<name>
vmauth-<name>-additional-service
```
——
```
  labels:
    app.kubernetes.io/component: monitoring
    app.kubernetes.io/instance: router
    app.kubernetes.io/name: vmauth
    managed-by: vm-operator
```

these labels are matching on the selector of vmauth’s vmservicescrape and vmagent is scraping duplicated endpoints with different job names.
It’s possible to add custom label selectors to serviceScrapeSpec and new labels to additional service to avoid targeting the additional service. but the default behavior of additional service and scraping duplication seems wrong.

**Comment by f41gh7** [comment]
For me, it looks like auto-created serviceScrape must have exclude matcher for additional service. I'll take a look for it.

**Comment by f41gh7** [comment]
Must be fixed v0.30.4 version
